### DISCUSSION TITLE
Extensions are ignored

### DISCUSSION THREAD

**Discussion by kimchirichie** [spatie/laravel-markdown#13] [General]
problem is two folds.

1. `MarkdownBladeComponent::class` does not use dependency injection to instantiate `$markdownRenderer`
2. `MarkdownRenderer@configureCommonMarkEnvironment` passes strings to`$environment->addExtension` whereas it expects `ConfigurableEnvironmentInterface`

**Comment by freekmurze** [comment]
Could you PR a fix for this with tests?
